library/std/src/collections/hash/set.rs

```diff
@@ -809,11 +809,11 @@ where
         other.is_subset(self)
     }
 
-    /// Adds a value to the set.
+    /// Inserts a value to the set.
```

Comment:
Grammatically I think it should be either "add..to" or "insert...into". I prefer the current wording.

Reply:
That's the least important change. I just changed it for consistency reasons after using "insert" multiple times.